.tekton/scanner-component-pipeline.yaml

```diff
@@ -236,6 +236,8 @@ spec:
       value: $(tasks.fetch-scanner-data.results.SOURCE_ARTIFACT)
     - name: CACHI2_ARTIFACT
       value: $(tasks.prefetch-dependencies.results.CACHI2_ARTIFACT)
+    - name: PLATFORM
+      value: linux/amd64
```

Comment:
[nit] Since we use non-remote buildah task for amd64, it ignores the `PLATFORM` parameter.
https://github.com/konflux-ci/build-definitions/tree/main/task/buildah-oci-ta/0.2

I suggest to omit it.

```suggestion
```